dot/types/block.go

```diff
@@ -65,6 +65,15 @@ func (b *Block) Encode() ([]byte, error) {
 	return append(enc, encBody...), nil
 }
 
+// MustEncode verifies that block is encoded
```

Comment:
could you update this to say "MustEncode returns the SCALE encoded block and panics if it fails to encode" or something along those lines? same with for header